Approve multiple candidates with a single signature (#1191)
paritytech/polkadot-sdk: polkadot/roadmap/implementers-guide/src/node/approval/approval-voting.md

aims of this subsystem.

Approval votes are split into two parts: Assignments and Approvals. Validators first broadcast their assignment to
indicate intent to check a candidate. Upon successfully checking, they don't immediately send the vote instead

> they queue the check for a short period of time `MAX_APPROVALS_COALESCE_TICKS` to give the opportunity of the

Reviewer: this is named MAX_APPROVALS_COALESCE_WAIT_TICKS in the code

    // this block. The block can be considered approved has all bits set to 1
    approved_bitfield: Bitfield,
    children: Vec<Hash>,

>     // A list of candidates that has been approved, but we didn't not sign and

Reviewer: ```suggestion
    // A list of candidates that we have checked, but didn't not sign and
```
